app/src/main/java/com/esri/arcgismaps/kotlin/sampleviewer/model/SampleMetadata.kt

```diff
@@ -8,16 +8,15 @@ import kotlinx.serialization.Serializable
  */
 @Serializable
 data class SampleMetadata(
-    @SerialName(value = "formal_name") val formalName: String,
-    @SerialName(value = "images") val imagePaths: List<String>,
-    @SerialName(value = "relevant_apis") val relevantApis: List<String>,
     @Serializable(with = SampleCategorySerializer::class)
-    @SerialName(value = "category") val sampleCategory: SampleCategory,
-    @SerialName(value = "redirectFrom") val redirectFrom: List<String>? = emptyList(),
+    @SerialName("category") val sampleCategory: SampleCategory,
     val description: String,
```

Comment:
Using the same serial name with the `val` name may be redundant, but I feel this would help ensure that the fields will be hard-checked if the metadata field names ever change. (Same for `keywords`,`title`)
```suggestion
    @SerialName("description") val description: String,
```

Reply:
It still is hard-checked by just the val description field